Client/Frontend/Components/ActionFooterView.swift

```diff
@@ -9,39 +9,40 @@ import ComponentLibrary
 struct ActionFooterViewModel {
     let title: String
     let actionTitle: String
-    let a11yTitleIdentifier: String = AccessibilityIdentifiers.ActionFooter.title
-    let a11yActionIdentifier: String = AccessibilityIdentifiers.ActionFooter.primaryAction
+    let a11yTitleIdentifier: String
+    let a11yActionIdentifier: String
+
+    init(title: String,
+         actionTitle: String,
+         a11yTitleIdentifier: String = AccessibilityIdentifiers.ActionFooter.title,
+         a11yActionIdentifier: String = AccessibilityIdentifiers.ActionFooter.primaryAction) {
+        self.title = title
+        self.actionTitle = actionTitle
+        self.a11yTitleIdentifier = a11yTitleIdentifier
+        self.a11yActionIdentifier = a11yActionIdentifier
+    }
 }
 
 final class ActionFooterView: UIView, ThemeApplicable {
     private struct UX {
         static let labelSize: CGFloat = 13
         static let buttonSize: CGFloat = 13
-        static let containerMargins = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
     }
 
     private lazy var titleLabel: UILabel = .build { label in
         label.font = DefaultDynamicFontHelper.preferredFont(
             withTextStyle: .footnote,
             size: UX.labelSize)
         label.numberOfLines = 0
-        label.setContentCompressionResistancePriority(.required, for: .vertical)
+        label.adjustsFontForContentSizeCategory = true
     }
 
     private lazy var primaryButton: ResizableButton = .build { button in
         button.titleLabel?.font = DefaultDynamicFontHelper.preferredFont(
             withTextStyle: .footnote,
             size: UX.buttonSize)
         button.buttonEdgeSpacing = 0
-        button.setContentCompressionResistancePriority(.required, for: .vertical)
-    }
-
-    private lazy var stackView: UIStackView = .build { stackView in
-        stackView.spacing = 0
```

Comment:
What was the reason for removing the stack view?

Reply:
It was breaking the layout when using the strings from the design.